hack/code/bandwidth_gen/main.go

```diff
@@ -83,26 +85,30 @@ func main() {
 				bandwidthData := row.FirstChild.NextSibling.NextSibling.NextSibling.FirstChild.Data
 				// exclude all rows that contain any of the following strings
 				if containsAny(bandwidthData, "Low", "Moderate", "High", "Up to") {
-					vagueBandwidth[instanceTypeData] = bandwidthData
+					vagueBandwidth[ec2types.InstanceType(instanceTypeData)] = bandwidthData
 					continue
 				}
 				bandwidthSlice := strings.Split(bandwidthData, " ")
 				// if the first value contains a multiplier i.e. (4x 100 Gigabit)
 				if strings.HasSuffix(bandwidthSlice[0], "x") {
 					multiplier := lo.Must(strconv.ParseFloat(bandwidthSlice[0][:len(bandwidthSlice[0])-1], 64))
-					bandwidth[instanceTypeData] = int64(lo.Must(strconv.ParseFloat(bandwidthSlice[1], 64)) * 1000 * multiplier)
+					bandwidth[ec2types.InstanceType(instanceTypeData)] = int64(lo.Must(strconv.ParseFloat(bandwidthSlice[1], 64)) * 1000 * multiplier)
 					// Check row for instancetype for described network performance value i.e (2 Gigabit)
 				} else {
-					bandwidth[instanceTypeData] = int64(lo.Must(strconv.ParseFloat(bandwidthSlice[0], 64)) * 1000)
+					bandwidth[ec2types.InstanceType(instanceTypeData)] = int64(lo.Must(strconv.ParseFloat(bandwidthSlice[0], 64)) * 1000)
 				}
 			}
 		}()
 	}
 	allInstanceTypes := getAllInstanceTypes()
 	instanceTypes := lo.Keys(bandwidth)
 	// 2d sort for readability
-	sort.Strings(allInstanceTypes)
-	sort.Strings(instanceTypes)
+	sort.SliceStable(allInstanceTypes, func(i, j int) bool {
+		return allInstanceTypes[i] < allInstanceTypes[j]
```

Comment:
Yeah that is exactly that is happening. It does work because ultimately they have a string value. I checked using some logs just to make sure it worked as expected.